portal-web/test/functional/com/liferay/portalweb/tests/enduser/exportimport/ExportImport.testcase

```diff
@@ -714,375 +836,441 @@ definition {
+			DataEngine.addField(
+				fieldFieldLabel = "Link to Page",
+				fieldName = "Link to Page");
 
-		WebContentStructures.saveCP(structureName = "WC Structure Name");
+			WebContentStructures.saveCP(structureName = "WC Structure Name");
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
+		task ("And: Add a new WC template related to the structure") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
 
-		NavItem.gotoTemplates();
+			NavItem.gotoTemplates();
 
-		WebContentTemplates.addCP(
-			structureName = "WC Structure Name",
-			templateDescription = "WC Template Description",
-			templateFieldNames = "Link to Page",
-			templateName = "WC Template Name");
+			WebContentTemplates.addCP(
+				structureName = "WC Structure Name",
+				templateDescription = "WC Template Description",
+				templateFieldNames = "Link to Page",
+				templateName = "WC Template Name");
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
+		task ("And: Add a new WC which is based on the WC template and structure and linked to the second page, then publish it") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
 
-		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
 
-		WebContent.addWithStructureCP(
-			structureName = "WC Structure Name",
-			templateName = "WC Template Name",
-			webContentLinkToPage = "Page B",
-			webContentTitle = "Web Content Title A");
+			WebContent.addWithStructureCP(
+				structureName = "WC Structure Name",
+				templateName = "WC Template Name",
+				webContentLinkToPage = "Page B",
+				webContentTitle = "Web Content Title A");
 
-		PortletEntry.publish();
+			PortletEntry.publish();
+		}
 
-		WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
+		task ("And: Add a new WC which is based on the WC template and structure and linked to the first page, then publish it") {
+			WebContentNavigator.openWebContentAdmin(siteURLKey = "site-a");
 
-		WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
+			WebContentNavigator.gotoAddWithStructureCP(structureName = "WC Structure Name");
 
-		WebContent.addWithStructureCP(
-			structureName = "WC Structure Name",
-			templateName = "WC Template Name",
-			webContentLinkToPage = "Page C",
-			webContentTitle = "Web Content Title B");
+			WebContent.addWithStructureCP(
+				structureName = "WC Structure Name",
+				templateName = "WC Template Name",
+				webContentLinkToPage = "Page C",
+				webContentTitle = "Web Content Title B");
 
-		PortletEntry.publish();
+			PortletEntry.publish();
+		}
 
-		for (var count : list "A,B") {
-			JSONLayout.addWidgetToPublicLayout(
-				groupName = "Site A",
-				layoutName = "Page ${count}",
-				widgetName = "Web Content Display");
+		task ("And: Add two WCD portlet to the first page then add a WC to each one") {
```

Comment:
Should be "And: Add WCD portlet to the first and second page and add a WC to each WCD"